runtests.sh

```diff
@@ -4,4 +4,4 @@ echo "Running flake8"
 flake8 || { echo "flake8 errors found!"; exit 1; }
 
 echo "Running Python tests"
-py.test tests/$* --cov-report html --cov treeherder
+DJANGO_SETTINGS_MODULE=tests.settings py.test tests/$* --cov-report html --cov treeherder
```

Comment:
I did it that way because it allows you to import wjatever needs django at the top of the module without raising an ImproperlyConfiguredException. The solution I had in mind was to use a pytest ini file and specify a custom ini option like they do in [pytest-django](https://github.com/pytest-dev/pytest-django/blob/4957b6f1efd004a6591976b86bee288055118962/pytest_django/plugin.py#L61).